server/middlewares/rateLimiter.ts

```diff
@@ -47,11 +66,18 @@ export function registerRateLimiter(config: RateLimiterConfig) {
     if (!RateLimiter.hasRateLimiter(ctx.path)) {
       RateLimiter.setRateLimiter(
         ctx.path,
-        defaults(config, {
-          duration: env.RATE_LIMITER_DURATION_WINDOW,
-          keyPrefix: RateLimiter.RATE_LIMITER_REDIS_KEY_PREFIX,
-          storeClient: Redis.defaultClient,
-        })
+        defaults(
+          {
+            ...config,
+            points: config.requests,
+          },
+          {
+            duration: 60,
```

Comment:
While the default duration and points is configurable by the end user for these hardcoded limits we need it to be a fixed value, otherwise it'll need to be defined on every route.

Reply:
Thanks :+1: I was about to suggest this because `points` didn't make any sense contextually.